ppx/browser/ppx_deriving_json_js.ml

```diff
@@ -65,13 +65,13 @@ module Of_json = struct
   let ensure_json_object ~loc x =
     [%expr
       if Stdlib.not [%e eis_json_object ~loc x] then
-        Ppx_deriving_json_runtime.of_json_error
+        Ppx_deriving_json_runtime.of_json_msg_error
           [%e estring ~loc (sprintf "expected a JSON object")]]
 
-  let ensure_json_array_len ~loc n len =
+  let ensure_json_array_len ~loc n len x =
     [%expr
       if Stdlib.( <> ) [%e len] [%e eint ~loc n] then
-        Ppx_deriving_json_runtime.of_json_error
+        Ppx_deriving_json_runtime.of_json_msg_error ~json:[%e x]
```

Comment:
@EmileTrotignon This function doesn't take a `json` arg but somehow the tests passed. 🤔

Reply:
fixed in https://github.com/melange-community/melange-json/pull/45/commits/3fa710dc65665e8541f4aa72d648af5dcb990f95 and https://github.com/melange-community/melange-json/pull/45/commits/be46a4d83110deb341f4b53ca4d1d4b37fc24e41